ensemble/ctf/editor.py

```diff
@@ -65,7 +65,7 @@ def perform_with_color(self, event, color):
 
 
 class AddOpacityAction(BaseCtfEditorAction):
-    name = 'Add Opacity'
+    name = 'Add Opacity Node...'
```

Comment:
But that's not an ellipsis! It's three periods! `</nitpick>`. (That is, if you're really serious about this, then I suspect that the text should include a single Unicode "HORIZONTAL ELLIPSIS" (U+22EE) codepoint rather than three "FULL STOP" (U+002E) codepoints.)